jenkins-agent.pkr.hcl

```
@@ -203,6 +225,18 @@ build {
     vm_size         = local.azure_vm_size
   }
 
+  provisioner "shell" {
+    # steps for docker images
+    only = ["docker.ubuntu-20"]
+    environment_vars = [
+      "DEBIAN_FRONTEND=noninteractive",
+    ]
+    #adding sudo and bash for docker software-properties-common 
```

```suggestion
    # adding sudo and bash for docker software-properties-common 
```
I restreined myself asking for an uppercase at the beginning of all comments